server/core/src/https/mod.rs

```diff
@@ -140,6 +140,7 @@ pub fn get_js_files(role: ServerRole) -> Result<JavaScriptFiles, ()> {
                 ("external/bootstrap.bundle.min.js", None, false, false),
                 ("external/htmx.min.1.9.12.js", None, false, false),
                 ("external/cred_update.js", None, false, false),
+                ("external/admin.js", None, false, false),
```

Comment:
probably shouldn't be in external, external means "sourced elsewhere". Should be in the root. 

Also I don't like extra js :P

Reply:
I'll try htmx OOB swaps